dev/domain-controller/FinalizeDeployment.ps1

```diff
@@ -1,10 +1,21 @@
 ﻿Param(
+  [string]$rgName,
   [string]$domainUserUPN,
   [string]$domainPassword,
   [string]$machineToJoin,
-  [string]$groupToJoin
+  [string]$groupToJoin,
+  [string]$azureUserName, 
+  [string]$azurePassword, 
```

Comment:
I followed the parameter domainUserUPN and domainPassword defined. the commandToExecute which will call this ps file  concats command and all parameters together, so not sure if we can use credential